src/memoize-one.ts

```diff
@@ -6,15 +6,22 @@ export type EqualityFn = (newArgs: any[], lastArgs: any[]) => boolean;
 export default function memoizeOne<
```

Comment:
okay, so this is now a generic that takes two arguments:

- `Args` which `extends unknown[]` (why not `extends any[]`?)
- `Result`

These are populated by

`resultFn: (this: any, ...newArgs: Args) => Result`

Reply:
> * which `extends unknown[]` (why not `extends any[]`?)

Either would be fine in this case. Generally speaking, I prefer `unknown` over `any` because the latter completely disables type checking.